docs/stories/18 - Sanctuary.adoc

```diff
@@ -0,0 +1,93 @@
+:doctype: book
+:icons: font
+:page-background-image: image:background_neutral.jpg[fit=fill, pdfwidth=100%]
+
+= Sanctuary
+Chris Garvey and Jeanne Kalvar
+
+image::story_images/18_sanctuary.png[sanctuary,fit=fill, pdfwidth=90%,align="center"]
+
+{empty} +
+
+{empty} +
+
+
+The Imperial ship, Memories of Jade, burst from the all-consuming mists, several dozen li from the shores of the hidden island the mists concealed. Their Sanctuary. One by one the other ships of the fleet emerged. This was it. The land Lady Doji promised, where they would be safe. Each of the Great Clans had prepared for this journey, ready to settle this new land.
+
+The clans were on their own to decide where they would make landfall for their new territory. No one knew what resources or dangers lie on this mysterious island. A poor choice could spell disaster for a clan. Overtures and threats, promises and blackmail, as complex an intrigue as ever had occurred in the Imperial Palace had brokered between the clans, carried in envoys traveling from one ship to another as they moved through the mists to the destination the Water Dragon had promised.
+
+Such promises meant little now.
+
+Each ship of the fleet seemed to pause either in reverence or awe at Sanctuary, but the moment was brief before each clan separated going their own way.
+
+'''
+
+{empty} +
```

Comment:
You don't need to add the empty stuff after every horizontal ruler. it was just that one time when it would have been attached too close to the text because it was directly at the top of the page. So, if it's not necessary (check the built pdf), fine. Just remove them all 🙂